### ISSUE TITLE
Issue about the setting of the code

### ISSUE THREAD

**Issue by ShileiCao** [Vibashan/online-od#7]
Is the provided code run in the online setting? Does this mean the model is first trained on the entire unlabeled target test set first and then evaluated on the same target test set? So it is a little different from the fully test-time adaption setting proposed by TENT.

**Comment by Vibashan** [comment]
Yes @ShileiCao, you are correct in your understanding. Our work is not fully test-time adaptation setting as proposed by TENT.
